### ISSUE TITLE
loss convergence curves

### ISSUE THREAD

**Issue by slala2121** [MiaoyunZhao/GANTransferLimitedData#2]
I've been getting loss curves like this during training though images appear to improve in quality across epochs. I'm curious how to interpret this against typical GAN loss curves. e.g., (https://stackoverflow.com/questions/42690721/how-to-interpret-the-discriminators-loss-and-the-generators-loss-in-generative) 

![Screen Shot 2020-11-18 at 8 20 08 PM](https://user-images.githubusercontent.com/58089750/99620939-77d5a980-29db-11eb-83fd-8f858546d198.png)

thanks

**Comment by MiaoyunZhao** [comment]
> I've been getting loss curves like this during training though images appear to improve in quality across epochs. I'm curious how to interpret this against typical GAN loss curves. e.g., (https://stackoverflow.com/questions/42690721/how-to-interpret-the-discriminators-loss-and-the-generators-loss-in-generative)
> 
> ![Screen Shot 2020-11-18 at 8 20 08 PM](https://user-images.githubusercontent.com/58089750/99620939-77d5a980-29db-11eb-83fd-8f858546d198.png)
> 
> thanks

The beginning of the curve may greatly depends on the initialization of the model. Since the model used here is initialized by a GAN model pretrained on ImageNet, which may different from the case in the link you have provided, the beginning of the loss curve looks different. Any way, both discriminator and generator losses (in your figure and the last figure in the link) are converging to some permanent numbers, which means the training is gradually stable. Also, I think the answer in the link you provided has good points on this problem. Thanks.

**Comment by slala2121** [comment]
ok, thanks!
